File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    words = ["aya", "ye", "woo", "ma"]
    for bab in babbling:
        w = bab
        for word in words:
            if word in w:
                w = w.replace(word, '*')
        w = w.replace('*', '')
        if w == '':
            answer += 1
        
    return answer

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        String[] words = {"aya", "ye", "woo", "ma"};
        for(String bob: babbling){
            String w = bob;
            for(String word: words){
                if(w.contains(word)){
                    w = w.replace(word, "*");
                }
            }
            if(w.contains("*")){
                w = w.replace("*", "");
            }
            if(w.isEmpty()){
                answer += 1;
            }
        }
        
        return answer;
    }
}